tests/equilibria/test_flux_surfaces.py

```diff
@@ -157,13 +163,68 @@ def setup_class(cls):
         eq_name = "eqref_OOB.json"
         filename = os.sep.join([TEST_PATH, eq_name])
         cls.eq = Equilibrium.from_eqdsk(filename)
+        cls.cache = {}
 
     def test_connection_length(self):
         flt = FieldLineTracer(self.eq)
         field_line = flt.trace_field_line(13, 0, n_points=1000)
         assert np.isclose(
             field_line.connection_length, field_line.coords.length, rtol=5e-2
         )
+        self.cache["fl_con_length_grid"] = field_line.connection_length
```

Comment:
This is a bit dangerous because tests can be run in any order or in isolation could we either combine the two tests or run the above twice, once as a test and once within the below test